doc/content/gateways/gateway-claiming/make-gateways-claimable.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Make a Gateway Claimable"
+description: ""
+weight: 1
+distributions: ["Enterprise", "Cloud"]
+--- 
+
+This guide explains the process of making a gateway claimable. When a gateway is claimable, someone else can claim it securely.
+
+<!--more-->
+
+## Prerequisites
+
+1. An gateway in The Things Stack Cloud. [See instructions]({{< ref "/gateways/adding-gateways" >}})
+
+## Authorize Claiming
+
+In order for anyone to claim a gateway that is owned by you, you need to authorize claiming. This is needed for {{% tts %}} to move the gateway to the other user. This needs to be done separately for each gateway.
+
+{{< cli-only >}}
+
+Replace `<gateway-id>` with the **Gateway ID** of the gateway that you created in prerequisites, and run the following command in the CLI:
```

Comment:
```suggestion
Replace the `<gateway-id>` with the **Gateway ID** of your gateway and run the following command in the CLI:
```

Reply:
```
Replace the `<gateway-id>`
```

This is grammatically incorrect. It's either 
```
Replace the `<gateway-id>` field (or value)
```

or (since this is already implied)
```
Replace `<gateway-id>`
```